Lib/hmac.py

```diff
@@ -50,7 +50,9 @@ def __init__(self, key, msg = None, digestmod = None):
 
         if digestmod is None:
             _warnings.warn("HMAC() without an explicit digestmod argument "
-                           "is deprecated.", PendingDeprecationWarning, 2)
+                           "is deprecated since Python 3.4, and will be removed "
```

Comment:
I'd be inclined to leave the warning text as is w/o mentioning version numbers there.  leave the versions to the docs.